claim/netdata-claim.sh.in

```diff
@@ -341,24 +341,29 @@ HERE_DOC
             exit $EXIT_CODE
         fi
 
+        if [ -z "${PROXY}" ]; then
+           PROXYMSG=""
+        else
+           PROXYMSG="You have claimed this node through a proxy - please update your the proxy setting in your netdata.conf to {$PROXY}. "
```

Comment:
Maybe change this `You have claimed this node through a proxy`  to something like `You have attempted claiming this node through a proxy`